ts/__tests__/Iterable.spec.ts

```diff
@@ -136,12 +136,21 @@ test("update user", () => {
   expect(MockRNIterableAPI.updateUser).toBeCalledWith(dataFields, false)
 })
 
-test("update email", () => {
+test("update email (no token)", () => {
   const newEmail = "woo@newemail.com"
 
   Iterable.updateEmail(newEmail)
 
-  expect(MockRNIterableAPI.updateEmail).toBeCalledWith(newEmail)
+  expect(MockRNIterableAPI.updateEmail).toBeCalledWith(newEmail, undefined)
+})
+
+test("update email (with token)", () => {
+  const newEmail = "woo@newemail.com"
+  const newToken = "token2"
+
+  Iterable.updateEmail(newEmail, newToken)
+
+  expect(MockRNIterableAPI.updateEmail).toBeCalledWith(newEmail, newToken)
 })
```

Comment:
So there are no tests for setEmail and setUserId with token?

Reply:
Yeah, not yet. I tried to set it up, but it was more complicated than expected.

Reply:
got it